### DISCUSSION TITLE
Support Timezone

### DISCUSSION THREAD

**Discussion by teddysbeach** [pocketbase/pocketbase#721] [Q&A] [answered]
Hi, I am using Pocketbase as my personal project thankfully.
I found out that the admin page from Pocketbase does support UTC timezone.
What I want to check is what time the users signed up with the local timezone (currently UTC only).
for that, I am wondering if you have plans for supporting the switching timezone on the admin panel.

Thanks for your great work!

![image](https://user-images.githubusercontent.com/64014774/193971985-87840815-f4a8-4abb-b7d5-2f958c54ec9f.png)
(pocketbase admin > users > created column )

**Comment by ganigeorgiev** [comment, marked as answer]
I'll consider showing the datetime value in the local timezone (and the actual stored date in a tooltip on hover), but this will be after v0.8.x because there will be a slight change to the returned datetime format (https://github.com/pocketbase/pocketbase/issues/472#issuecomment-1255056640).

**Reply by teddysbeach** [reply]
Thanks for letting us know! Have a good day!
